vagrant-post-script/orchestrate_docker.sh

```diff
@@ -16,3 +16,10 @@ if [[ -f .init ]]; then
 else
   make bootstrap
 fi
+
+
+# frontend helper
+cd frontend || exit 1
+echo "node: $(node --version)"
+echo "npm: $(npm --version)"
+npm i || npm i --force && npx vue-cli-service build && chmod 775 -R dist && docker restart myaegee_frontend_1
```

Comment:
```suggestion
npm i || npm i --force && npm run build && chmod 775 -R dist && docker restart myaegee_frontend_1
```

What does `npm i || npm i --force` do? Does it try `npm i` first and if that fails use `npm i --force`?

Reply:
correct. i kept using npx because all of cristian's problems are not encountered here (damn), what do you think should be used?

Reply:
The `npx` one doesn't work with Node 18 so we should use `npm run build`. Before it was both executing the same command

Reply:
BUT we do use node 16 inside vagrant (see prerequisites.sh)

Reply:
We can update that btw. All submodules use Node 18 now

Reply:
iirc there was a reason for using 16, given that this pinning happened only 2 months ago. Of course I do not remember this reason :))) if you say to put 18, we put 18

Reply:
That was because frontend did not work with Node 18 yet, but that was fixed last week. We should be good for Node 18

Reply:
the pain never ends https://stackoverflow.com/questions/74815894/install-nodejs-v18-on-ubuntu-18-04

Reply:
Maybe we can have Ubuntu 20.04 (or 22.04) locally already? I used 20.04 for my own server and that works great, now I am on 22.04 but I didn't do much on it yet

Reply:
I would keep it for a separate MR